modules/developers-guide/pages/existing_cstar.adoc

```diff
@@ -0,0 +1,31 @@
+= Installing Stargate with an existing Cassandra cluster
+
+Stargate can be installed alongside an existing Cassandra cluster.
+
+== Using Stargate with Cassandra 3.11
+
+include::quickstart:page$quick_start.adoc[tag=getDockerImage,leveloffset=+1]
+
+=== Starting Stargate with Cassandra 3.11
+
+To join a running Cassandra 3.11 cluster:
+
+[source,bash,subs="attributes+"]
+----
+docker run --name stargate \
+  -d stargateio/stargate-3_11:{stargate-docker-tag} \
+  --cluster-name stargate \
+  --cluster-seed 127.0.0.1 \
+  --cluster-version 3.11 \
+  --listen 127.0.0.2 \
+  --simple-snitch
```

Comment:
Has this been tested on OSX? I'm concerned how 127.0.0.1 will be resolved since that will be localhost of the container rather than the host.